.github/workflows/push.yml

```
@@ -39,27 +39,78 @@ jobs:
       - name: Compile Application
         uses: splunk-soar-connectors/.github/.github/actions/compile-app@main
 
-  sanity-test:
-    runs-on: ubuntu-latest # This will be Codebuild
+  test-setup:
+    runs-on: ubuntu-latest
     needs: [build, compile]
+    outputs:
+      fips_compliant: ${{ steps.set-outputs.outputs.fips_compliant }}
+      publisher: ${{ steps.set-outputs.outputs.publisher }}
     steps:
-      - name: Setup Environment
-        uses: splunk-soar-connectors/.github/.github/actions/env-setup@main
+      - name: Check out actions repo
+        uses: actions/checkout@v4
+      
+      - name: Set sanity tests enviornment variables
+        uses: splunk-soar-connectors/.github/.github/actions/test-setup@tapishj/PAPP-35403
+      
+      - name: Set outputs
+        id: set-outputs
+        run: |
+          echo "publisher=${{ env.publisher }}" >> $GITHUB_OUTPUT
+          echo "fips_compliant=${{ env.fips_compliant }}" >> $GITHUB_OUTPUT
+
+  sanity-test:
+    runs-on: 
+      - codebuild-integration-tests-${{ github.run_id }}-${{ github.run_attempt }}
+      - image:custom-linux-875003031410.dkr.ecr.us-west-2.amazonaws.com/soar-connectors/pytest:f7150dbb7f347d35f8f4bb285d36985ecd4cf231
+    needs: [build, compile, test-setup]
+    strategy:
+      matrix:
+        include:
+          - version: "current"
+            ip: ${{ vars.PHANTOM_INSTANCE_CURRENT_VERSION_IP }}
+          - version: "next"
+            ip: ${{ vars.PHANTOM_INSTANCE_NEXT_VERSION_IP }}
+          - version: "previous"
+            ip: ${{ vars.PHANTOM_INSTANCE_PREVIOUS_VERSION_IP }}
+          - version: "cloud"
+            ip: ${{ vars.PHANTOM_INSTANCE_CLOUD_HOST }}
+          - version: "rhel"
+            ip: ${{ vars.PHANTOM_INSTANCE_CURRENT_RHEL_VERSION_IP }}
+    env:
+      APP_REPO: ${{ github.event.repository.name }}
+      AWS_PHANTOM_INSTANCE_CURRENT_VERSION_IP: ${{ vars.AWS_PHANTOM_INSTANCE_CURRENT_VERSION_IP }}
+      AWS_PHANTOM_INSTANCE_PREVIOUS_VERSION_IP: ${{ vars.AWS_PHANTOM_INSTANCE_PREVIOUS_VERSION_IP }}
+      NUM_TEST_RETRIES: ${{ vars.NUM_TEST_RETRIES }}
+      VAULT_ROLE: ${{ vars.VAULT_ROLE }}
+      PHANTOM_INSTANCE_CURRENT_FIPS_VERSION_IP: ${{ vars.PHANTOM_INSTANCE_CURRENT_FIPS_VERSION_IP }}
+
+    steps:
+      - name: Check out actions repo
+        uses: actions/checkout@v4
```

I don't think this actually checks out the "actions" repo. Maybe just remove the name here?
```suggestion
      - uses: actions/checkout@v4
```